model/project.go

```diff
@@ -1693,9 +1700,6 @@ func (p *Project) ResolvePatchVTs(patchDoc *patch.Patch, requester, alias string
 	if len(tasks) == 1 && tasks[0] == "all" {
 		tasks = []string{}
 		for _, t := range p.Tasks {
-			if !utility.FromBoolTPtr(t.Patchable) || utility.FromBoolPtr(t.GitTagOnly) {
```

Comment:
Note that this check is not actually necessary (and also has a bug because it ignores precedence rules for these two fields). Checking for a matching requester on L1726 performs the same check while respecting precedence.